### ISSUE TITLE
Review yazi.nvim options

### ISSUE THREAD

**Issue by mikavilpas** [NormalNvim/NormalNvim#498]
Since the release of yazi 0.3.1, some things have changed in yazi.nvim as well (https://github.com/mikavilpas/yazi.nvim/blob/main/CHANGELOG.md):

- yazi 0.3 is now required (this is probably fine for most users)
- the `use_yazi_client_id_flag` option was removed (no longer required but causes no issues)
- the `use_ya_for_events_reading` option was removed for the same reason (no longer required)

So in summary, no changes are required but it's good to be aware of 🙂 If you want to, you can remove the deprecated options in NormalNvim too.

**Comment by Zeioth** [comment]
Removed deprecated options in `v4.0.0 RC7`!
https://github.com/NormalNvim/NormalNvim/releases/tag/v4.0.0-rc7
